src/update.py

```diff
@@ -176,8 +189,12 @@ def _get_db_engine(args):
 
     os.environ["NLS_LANG"] = ".AL32UTF8"
 
-    connection_str = "oracle+cx_oracle://{0}:{1}@{2}:{3}/{4}".format(
-        args.db_user, args.db_pass, args.db_host, args.db_port, args.db_name)
+    if args.db_type == 'oracle':
+        connection_str = "oracle+cx_oracle://{0}:{1}@{2}:{3}/{4}".format(
```

Comment:
Similar comment regarding the connection string as above. It may be better to have a common definition for the connection string before the if/else, then to substitute the first part based on the specific database.

That said, it's not clear to me whether all DBs would have a common connection string format, so perhaps this doesn't make sense.